src/app/components/mnoe-users-local-list/mnoe-users-local-list.html

```diff
@@ -25,7 +27,9 @@
             <a href class='email_edit_link' ng-hide="user.isUpdatingEmail || emailForm.$visible" ng-click="emailForm.$show()"><i class="fa fa-pencil-square-o fa-1"></i></a>
             <span ng-show="user.isUpdatingEmail"><i class="fa fa-spinner fa-pulse fa-fw"></i></span>
           </td>
+          <td>{{user.uid}}</td>
           <td>{{user.created_at | date: 'dd/MM/yyyy'}}</td>
+          <td>{{user.last_active_at || 'mnoe_admin_panel.dashboard.users.widget.local_list.last_login.never' | translate }}</td>
```

Comment:
We should probably use a date filter here just like we have for `user.created_at`